var/spack/repos/builtin/packages/superlu-dist/package.py

```diff
@@ -88,7 +88,8 @@ def install(self, spec, prefix):
             'LOADER       = {0}'.format(self.spec['mpi'].mpif77),
             'LOADOPTS     =',
             'CDEFS        = %s' % ("-DNoChange"
-                                       if '%xl' in spec or '%xl_r' in spec
+                                       if spack_f77.endswith('xlf') or
```

Comment:
p.s. this will always be the case for 
```
`%xl` and `%xl_r` in spec`
```
right? In other words, you just extend the domain where `if` applies?

Reply:
Correct